static/templates/user_presence_rows.handlebars

```diff
@@ -2,7 +2,7 @@
 {{#each starred_users}}
 {{partial "user_presence_row"}}
 {{/each}}
-<hr>
+<hr/>
 {{#each unstarred_users}}
 {{partial "user_presence_row"}}
 {{/each}}
```

Comment:
This commit can be merged with https://github.com/zulip/zulip/pull/850/commits/681b057b2de703daa00b68e2f79771c8e44b9ddb